### PR TITLE
Update for Trakt to allow FanartHandler paths for Clearart in movies

### PR DESCRIPTION
**Pull request by KJ-Cox**
Trakt.Common.Movies.xml and Trakt.Popular.Movies.xml modified to use FanartHandler paths.

Trakt Movie xml files added to add FanartHandler hack to enable Clearart.
This is the tag that gets read in the xml files: 
Changed plugin icon to movie icon.

This requires FanartHandler update to work.

### CODE REVIEW

**Comment by andrewjswan** [comment]
Merge after we update FH.

**Comment by andrewjswan** [comment]
Looks great.

File: Common/TVSeries.ArtworkChooser.xml [hunk 1]
```diff
@@ -152,7 +152,7 @@
    		<posY>150</posY>
    		<width>250</width>
    		<height>140</height>
-   		<texture>..\..\..\Thumbs\ClearArt\Series\#(string.trim(#TVSeries.Series.ID)).png</texture>
+   		<texture>#fanarthandler.series.clearart.selected</texture>
```
Comment (andrewjswan) [inline Common/TVSeries.ArtworkChooser.xml:155]: Need to change from `#fanarthandler.series......` to `#fanarthandler.tvseries.....`

Comment (KJ-Cox) [reply Common/TVSeries.ArtworkChooser.xml]: Completed

**Pull request merged**